python/src/ops.cpp

```diff
@@ -4401,6 +4401,7 @@ void init_ops(nb::module_& m) {
         Args:
             a (array): Input array or scalar.
             scale (float): Scale the output by this factor.
+              Defaults to `1/sqrt(a.shape[-1])` so that H is orthonormal.
```

Comment:
```suggestion
              Defaults to ``1/sqrt(a.shape[-1])`` so that the Hadamard matrix is orthonormal.
```

I think `H` doesn't have context here.